.github/ISSUE_TEMPLATE/enhancement.yml

```diff
@@ -3,24 +3,27 @@ description: Propose an enhancement for Babelfish for PostgreSQL Website
 title: "[Enhancement]: "
 labels: ["enhancement", "untriaged"]
 assignees:
-  - amazon-auto
+  - 3manuek
+  - gopinathpai
+  - xaviersierra
+  - nasbyj
 body:
   - type: markdown
     attributes:
       value: |
-        Thanks for taking the time to fill out this bug report!
+        Thanks for taking the time to fill out this feature request.
   - type: input
     id: contact
     attributes:
       label: Contact Details
-      description: How can we get in touch with you if we need more info?
+      description: Please enter your mail address for contacting in the case of needing more information.
```

Comment:
How about:

Provide an email address where we can contact you if we need more information.

Reply:
Removed Contact Details section.